runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -174,6 +178,39 @@ public void deleteTable(String namespace, String tableName) {
         runtime.getTableRegistry().deleteTable(namespace, tableName);
     }
 
+    /**
+     * Appends an empty checkpoint to the given trimmed table. After this is done,
+     * it allows to reopen and access a trimmed table for limited period of time
+     * until the next cycle of prefix trim happens. The main use case of this method
+     * is the auto-remediation process for trimmed tables in openTable.
+     *
+     * @param namespace the namespace of the table
+     * @param tableName the name of the table
+     */
+    public void resetTrimmedTable(String namespace, String tableName) {
+        String table = TableRegistry.getFullyQualifiedTableName(namespace, tableName);
+        UUID streamId = CorfuRuntime.getStreamID(table);
```

Comment:
since this is a public api method, can we please add logging to indicate that a trimmed table is being reset?